pkgs/tools/security/sudo/default.nix

```
@@ -58,7 +65,7 @@ stdenv.mkDerivation rec {
 
   postInstall =
     ''
```

```suggestion
  postInstall = ''
```
when we are reformatting we should fix this, too.